frontend/src/pages/AccountSend.vue

```diff
@@ -431,6 +761,22 @@ function useSendForm() {
         await setNetwork(chain[0]);
       }
     }
+
+    batchSendIsSupported.value = batchSendSupportedChains.includes(Number(linkChainId));
```

Comment:
Same comment. But also, seems weird that we're doing this inside `setPaymentLinkData`? They feel unrelated. Right now this logic runs twice on mount because we call `setPaymentLinkData` which runs this line, then this same line exists directly in `onMounted`

Reply:
Looks like a duplicate leftover from me trying to fix the flakiness around showing/hiding batchSend when switching networks. Removed the line from `setPaymentLinkData` and everything is still working as expected. https://github.com/ScopeLift/umbra-protocol/pull/491/commits/85765a48270e8cb1953f8fa41932a7dce1544fe2